src/KenwardRoger.jl

```diff
@@ -4,5 +4,112 @@ using CSV
 using DataFrames
 using ForwardDiff
 using MixedModels
+using Distributions
+using LinearAlgebra
+
+include("struct.jl")
+
+export kenwardroger_matrices, kenwardroger_estimates
```

Comment:
I prefer only exporting a single function, the end user has little use for the matrices.